IronPython3Extension/Extension.cs

```diff
@@ -29,9 +35,78 @@ public void Dispose()
             
         }
 
-        public void Ready(ReadyParams sp)
+        /// <summary>
+        /// Action to be invoked when the Dynamo has started up and is ready
+        /// for user interaction. 
+        /// </summary>
+        /// <param name="rp"></param>
+        public void Ready(ReadyParams rp)
+        {
+            var extraPath = Path.Combine(new FileInfo(Assembly.GetExecutingAssembly().Location).Directory.Parent.FullName, "extra");
+            var alc = new IsolatedPythonContext(Path.Combine(extraPath, $"{PythonEvaluatorAssembly}.dll"));
+            var dsIronAssem = alc.LoadFromAssemblyName(new AssemblyName(PythonEvaluatorAssembly));
+
+            //load the engine into Dynamo ourselves.
+            LoadPythonEngine(dsIronAssem);
+
+            //we used to do this:
+            //but it's not neccesary to load anything into the VM.
+            //instead we skip all the extra work and trigger the side effect we want
+            //which is re executing the graph after the dsIronPython evaluator is loaded into the PythonEngineManager.
+            //rp.StartupParams.LibraryLoader.LoadNodeLibrary(dsIronAssem);
```

Comment:
Why do we need to re-execute the graph again?

Reply:
This was to fix a bug @avidit filed - before using assembly load contexts, our use of LoadLibrary had the side effect of re executing the graph, so when  a user installed the IronPython package, the nodes would get resolved and then run, resolving errors.

With ALC we're not using LoadLibrary to inject the types anymore, so we have to kick off a run manually to get that same behavior.